### ISSUE TITLE
salt encryption

### ISSUE THREAD

**Issue by fgibaux** [ssmulders/hashed-passport#4]
Hi,

I noticed that the latest change to the salt encryption 69f4fa4 breaks the package as the `encrypt` function doesn't seem to have a constant output (I guess there is some random salt involved in the encryption).

**Comment by ssmulders** [comment]
Good catch, I overcomplicated things. Changing it to use the `APP_NAME` as the salt.

The idea behind using encrypt was so the APP_KEY couldn't be reversed from the hashed values. But figuring out the APP_NAME is fine, so it's unique enough to use as a salt but poses no security risk.

Deploying a patch right away :)

**Comment by ssmulders** [comment]
Here you go: https://github.com/ssmulders/hashed-passport/releases/tag/1.1.1

Thanks @fgibaux  !

**Comment by fgibaux** [comment]
Thanks for the reactivity ! Actually was not that urgent because the commit wasn't yet in the release.

Now I'm wondering if APP_NAME is a good choice. Also I guess changing the origin of the salt breaks all the current implementation as the hashed id changed. :confused: 

Maybe a config value of some sort, with the default as APP_KEY to remain backward compatible, while still allowing people to use something more safe?

**Comment by ssmulders** [comment]
Thanks for the feedback @fgibaux ! This is my first public repo, and first package that I released. It seems I had some reading up to do on how to best approach things ;-) I'll delete the bug fix release (as, like you said, the commit wasn't in release yet) and then setup things properly this time around. 

If you're up for it - I have added you as a repo collaborator - would be great to have someone with a bit more experience in the repo field as a second pair of eyes.
